components/Buttons/UserButton.vue

```diff
@@ -42,14 +38,32 @@ export default Vue.extend({
   },
   computed: {
     menuItems(): MenuItem[] {
-      return [
-        {
-          title: this.$t('logout'),
+      const menuItems = [];
+
+      if (this.$auth.$state.user.Policy.IsAdministrator) {
+        menuItems.push({
+          title: this.$t('metadata'),
           action: (): void => {
-            this.logoutUser();
+            this.$router.push('/metadata');
           }
+        });
+      }
+
+      menuItems.push({
+        title: this.$t('settings'),
+        action: (): void => {
+          this.$router.push('/settings');
```

Comment:
Same with settings, for user settings, it makes sense, but for server settings? not so much.

Reply:
There's not really a distinction between "server" and "client" settings pages anymore. It's "Jellyfin settings", basically, so that's our main and only settings place.

As such, it makes sense for it to be there, since that's what users would expect, based on other services.